examples/runDecamQuickTest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+print_error() {
+    >&2 echo "$@"
+}
+
+PRODUCT_DIR=${VALIDATE_DRP_DIR}
+# OS X El Capitan SIP swallows DYLD_LIBRARY_PATH so export the duplicate in LSST_LIBRARY_PATH
```

Comment:
Is it necessary to do this here? It's just going to get removed from the environment again when you execute `runExample.sh`; you go through this whole dance again there.